### PR TITLE
Some layout improvements and yup validation implementation

### PR DESCRIPTION
**Pull request by stdavis**
The layout improvements aren't perfect, but I do think that they are headed in the right direction.

Here's an example of some validation messages:
image

### CODE REVIEW

File: src/explorer/README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ This app runs as a docker container. To build the image `docker-compose build ex
 
 In the project directory, you can run:
 
+#### `npm run build:full-css`
+
+- creates `src/styles/tailwind.css`
+- do this before running `npm start` for the first time
+
```
Comment (steveoh) [inline src/explorer/README.md:23]: **question**: should this be a npm start hook or something?

Comment (stdavis) [reply src/explorer/README.md:23]: Probably. I haven't yet researched the two different build css tasks and wasn't really clear on the difference between them where.

File: src/explorer/src/components/Endpoint/Geocoding/StreetZone/StreetZoneApi.js [hunk 2]
```diff
@@ -34,21 +34,17 @@ export default function StreetZone(props) {
 
   return (
     <>
-      <EndpointInput name="street" {...defaultAttributes.street} dispatch={dispatch} />
-      <EndpointInput name="zone" {...defaultAttributes.zone} dispatch={dispatch} />
+      <EndpointInput name="street" schema={schema.fields.street.clone()} dispatch={dispatch} />
```
Comment (steveoh) [inline src/explorer/src/components/Endpoint/Geocoding/StreetZone/StreetZoneApi.js:37]: **question**: did you learn that `clone` is required? What happens if it isn't there?

File: src/explorer/src/components/Endpoint/Geocoding/StreetZone/meta.js [hunk 3]
```diff
@@ -1,61 +1,32 @@
...
-};
+const schema = object().shape({
+  street: string().matches(/^\d*\s\D/, {
+    message: 'Must be valid street address',
+    excludeEmptyString: true
+  }).required().meta({ placeholder: '123 south main street'}),
+  zone: string().matches(/(^84\d{3}$|^\D)/, {
+      message: 'Must be valid Utah zip code or city name',
+      excludeEmptyString: true
+    }).required().meta({ placeholder: 'SLC or 84111' }),
```
Comment (steveoh) [inline src/explorer/src/components/Endpoint/Geocoding/StreetZone/meta.js:11]: **nitpick**: the default yup schema messages are lowercase following a `this is` or `this must` template. I think we should match that for consistency. 
```suggestion
  street: string().matches(/^\d*\s\D/, {
    message: 'this must be a valid street address',
    excludeEmptyString: true
  }).required().meta({ placeholder: '123 south main street'}),
  zone: string().matches(/(^84\d{3}$|^\D)/, {
      message: 'this must be a valid Utah zip code or city name',
      excludeEmptyString: true
    }).required().meta({ placeholder: 'SLC or 84111' }),
```

**Pull request merged**